aries_vcx/src/lib.rs

```diff
@@ -40,6 +40,10 @@ pub mod global;
 pub mod indy;
 pub mod protocols;
 
+pub mod core;
+pub mod plugins;
+pub mod xyz;
```

Comment:
@Patrik-Stas looking for suggestions on what to call this - lol. As discussed, `xyz` contains a lot of methods previously from `indy` but made to be generic for `Profile`s, not indy handles. They are helper functions and impls which aggregate several calls to profiles

Reply:
lets do just boring `common` / `misc`

Reply:
renamed to `common`